src/docs/cookbook-payments/removing-stripe/index.md

```diff
@@ -0,0 +1,332 @@
+---
+title: How to remove Stripe and payments
+slug: removing-stripe-and-payments
+updated: 2020-07-06
+category: cookbook-payments
+ingress:
+  In some cases, you might want to remove Stripe integrations or even all the
+  payments from FTW. This article gives you a starting point for these
+  customizations.
+published: true
+---
+
+# Removing Stripe
+
+In this guide, we will go through the minimum changes needed to make the booking
+flow work without Stripe. Most likely you need to customize some parts (e.g.
+transaction process, email templates, checkout page) more depending on your
+marketplace idea. You might also want to remove e.g. unused Stripe components
+from your project to clean up the code.
+
+###1. Update the transaction process and remove Stripe related actions
+
+You need to remove at least the following actions:
+
+- :action/stripe-create-payment-intent
+- :action/stripe-capture-payment-intent
+- :action/stripe-confirm-payment-intent
+- :action/stripe-create-payout
+- :action/stripe-refund-charge
+- :action/stripe-refund-payment
+
+If you are using the FTW default process with strong customer authentication
+(e.g. flex-default-process) you need to remove confirming the payment.
+Otherwise, the transaction will get stuck. The simplest way to do this is to
+remove `pending-payment` and `expire-payment` states and point `request-payment`
+and `equest-payment-after-enquiry` directly to `preauthorized` state. However,
+when you are building the transaction process to your marketplace it might make
+sense to rename some of the states and transitions (e.g. preauthorized) so that
+they make more sense in your use-case.
```

Comment:
```suggestion
sense to rename some of the states and transitions so that
they make more sense in your use-case. E.g. *preauthorized* state refers to the state of payment - it is preauthorized to be captured.
```